### DISCUSSION TITLE
No Recognition

### DISCUSSION THREAD

**Discussion by Pilgrim90** [aclap-dev/video-downloadhelper#1123] [Help and Q&A]
### Your question

I loaded Downloadhelper like a week ago again, after a breakdown of my laptop. Now it does not recognize any video, does not let me download a video despite having the Downloader - app and all.
Please help me out.

### Addon & CoApp version and details

I loaded Downloadhelper like a week ago again, after a breakdown of my laptop. Now it does not recognize any video, does not let me download a video despite having the Downloader - app and all.
Please help me out.

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

If you want help from somebody else, first you have to help yourself.  You must supply a proper problem description.  What you have posted above is not a proper problem description.  The section labelled `Addon & CoApp version and details` must be filled in.  There were instructions on the web page you used to create your problem report.  You ignored them.  You need to follow them.  You can find those instructions again by clicking the Wiki link that appears at the top of every page of this forum.  That takes you to the closest thing we have to a user manual.  You need to get in the habit of referring to that.  It shows a table of contents down the right side.  At the bottom of the table of contents there is a link that reads `Show 8 more pages...`.  Click that.  That reveals more entries in the table of contents.  Click the one that reads `Where can I find my versions?`  Follow those instructions.

You also need to supply at least one URL of a video that you are having trouble with.  Don't tell me that it's every video.  If I tell you that I am able to download every video I want with VDH, how much does that help you?  In the same way, telling me that every video you try fails to download in VDH is an entirely unhelpful statement.  You're not helping yourself because that's not giving me any clues what you are trying to do.  So you have to give me at least one URL of a video that is causing you problems.  A URL.  Not the name of a web site.  The URL of a specific video.  It has to be on a web site that does not require me to register a user ID & password before I can look at the video.  But you have to give the URL of a specific video.  Do not just tell me a web site.  I will not scrounge around on some web site I've probably never heard of looking for examples.  You have examples.  Post at least one of those URLs.

Post all of that here by modifying your post above.  Look above your post to the right.  You will see `...` 3 dots.  Click that.  I think you will be able to take it from there.  Once you correct your post so it's a proper problem report, we can try to get to the bottom of your problem.

**Reply by Pilgrim90** [reply]
It is with everyvideo.
